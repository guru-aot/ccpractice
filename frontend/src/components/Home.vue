<template>
<div>
 <b-navbar toggleable="lg" type="dark" variant="info">
    <b-navbar-brand href="/">Logo</b-navbar-brand>
    <b-navbar-toggle target="nav-collapse"></b-navbar-toggle>

    <b-collapse id="nav-collapse" is-nav >
      <b-navbar-nav class="ml-auto">
        <b-nav-item href="/">Home</b-nav-item>
        <b-nav-item href="/dashboard">Login</b-nav-item>
      </b-navbar-nav>
    </b-collapse>
  </b-navbar>
</div>
</template>
<script>
export default {
  name: 'Home',
  props: {
    msg: String,
  },
};
</script>
<style scoped>
.bg-info {
    background-color: #003366 !important;
}
</style>
