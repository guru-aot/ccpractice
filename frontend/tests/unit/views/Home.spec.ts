import { shallowMount } from '@vue/test-utils';
import Home from '@/views/Home.vue';

describe('Home.vue', () => {
  it('renders without crashing', () => {
    const wrapper = shallowMount(Home);
    expect(wrapper.element).toMatchSnapshot();
  });
});
