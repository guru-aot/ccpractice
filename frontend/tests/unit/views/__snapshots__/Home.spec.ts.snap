// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue renders without crashing 1`] = `<helloworld-stub />`;
